homeassistant/components/media_player/webostv.py

```diff
@@ -44,11 +44,45 @@
 WEBOS_APP_LIVETV = 'com.webos.app.livetv'
 WEBOS_APP_YOUTUBE = 'youtube.leanback.v4'
 WEBOS_APP_MAKO = 'makotv'
+WEBOS_APP_NOWTV = 'now.tv'
+WEBOS_APP_NETFLIX = 'netflix'
+WEBOS_APP_AMAZONPRIME = 'lovefilm'
+WEBOS_APP_IPLAYER = 'bbc.iplayer.3.0'
+WEBOS_APP_ITV = 'com.fvp.itv'
+WEBOS_APP_4OD = 'com.fvp.ch4'
+WEBOS_APP_GOOGLEPLAY = 'googleplaymovieswebos'
+WEBOS_APP_TVGUIDE = 'com.webos.app.tvguide'
+WEBOS_APP_MIRACAST = 'com.webos.app.miracast'
+WEBOS_APP_TVGUIDE = 'com.webos.app.tvguide'
+WEBOS_APP_BROWSER = 'com.webos.app.browser_default'
+WEBOS_APP_MULTIVIEW = 'com.webos.app.multiview'
+WEBOS_APP_PLEX = 'cdp-30'
+WEBOS_APP_BBCNEWS = 'bbc.news.3.0'
+WEBOS_APP_HDMI1 = 'com.webos.app.hdmi1'
+WEBOS_APP_HDMI2 = 'com.webos.app.hdmi2'
+WEBOS_APP_HDMI3 = 'com.webos.app.hdmi3'
+
 
 WEBOS_APPS_SHORT = {
     'livetv': WEBOS_APP_LIVETV,
     'youtube': WEBOS_APP_YOUTUBE,
-    'makotv': WEBOS_APP_MAKO
+    'makotv': WEBOS_APP_MAKO,
+    'nowtv': WEBOS_APP_NOWTV,
+    'netflix': WEBOS_APP_NETFLIX,
+    'amazonprime': WEBOS_APP_AMAZONPRIME,
+    'iplayer': WEBOS_APP_IPLAYER,
+    'itvplayer': WEBOS_APP_ITV,
+    '4od': WEBOS_APP_4OD,
+    'googleplaymovies': WEBOS_APP_GOOGLEPLAY,
+    'tvguide': WEBOS_APP_TVGUIDE,
+    'miracast': WEBOS_APP_MIRACAST,
+    'browser': WEBOS_APP_BROWSER,
+    'multicast': WEBOS_APP_MULTIVIEW,
+    'plex': WEBOS_APP_PLEX,
+    'bbcnews': WEBOS_APP_BBCNEWS,
+    'hdmi1': WEBOS_APP_HDMI1,
+    'hdmi2': WEBOS_APP_HDMI2,
+    'hdmi3': WEBOS_APP_HDMI3
```

Comment:
I can see how the list can help for apps with strange names like plex that it's app name is cdp-30.
can you remove the HDMI strings?  they are added automatically if they are active (if something is connected)